tests/test_filter.py

```diff
@@ -165,7 +165,7 @@ def test_include_and_exclude(include, exclude, expected_files):
             num_workers=pytest.NUM_WORKERS,
             verbose=False,
         )
-    assert list(db.files) == expected_files
+    assert sorted(list(db.files)) == expected_files
```

Comment:
The other option would be not not sort and change `expected_files` acordingly.

Reply:
I did this before, but then we have to depend on `audformat>=0.13.0`. With the curent solution it also works with older `audformat` versions.

Reply:
Mhh, I still think it's probably not a bad idea to force `audformat>=0.13.0`..

Reply:
OK, I adjusted the tests and `audformat` dependencies.

But do you think we should change the tests here, or make a new `audformat` release and sort the output of `db.files` and `db.segments`?

Reply:
I think, I would prefer if `db.files` and `db.segments` still return their result in a sorted way.

Reply:
Mhh, good point. In fact for `db.files` and `db.segments` it indeed could make sense to sort them as they possibly come from several tables anyway and I don't think the user has a expectation that a certain order is kept.

Reply:
Good, I created https://github.com/audeering/audformat/pull/149